pages/_app.tsx

```diff
@@ -87,7 +87,7 @@ export default class App extends NextApp {
           {/* <script src="https://cdn.trackjs.com/agent/v3/latest/t.js"></script> */}
           
         </Head>
-        <Script src="/jquery.js" />
+        <Script src="/jquery.js" strategy={'beforeInteractive'}/>
```

Comment:
without this change, when running locally, i constantly got errors about jquery methods and objects